Source/Particles/Gather/FieldGather.H

```diff
@@ -379,18 +379,18 @@ void doGatherShapeN (const amrex::ParticleReal xp,
 /**
  * \brief Field gather for particles
  *
- * /param getPosition          : A functor for returning the particle position.
- * /param getExternalEField    : A functor for assigning the external E field.
- * /param getExternalBField    : A functor for assigning the external B field.
- * \param Exp, Eyp, Ezp        : Pointer to array of electric field on particles.
- * \param Bxp, Byp, Bzp        : Pointer to array of magnetic field on particles.
- * \param exfab eyfab ezfab    : Array4 of the electric field, either full array or tile.
- * \param ezfab bxfab bzfab    : Array4 of the magnetic field, either full array or tile.
- * \param np_to_gather         : Number of particles for which field is gathered.
- * \param dx                   : 3D cell size
- * \param xyzmin               : Physical lower bounds of domain.
- * \param lo                   : Index lower bounds of domain.
- * \param n_rz_azimuthal_modes : Number of azimuthal modes when using RZ geometry
+ * \param getPosition          A functor for returning the particle position.
```

Comment:
```suggestion
 * \tparam depos_order         deposition order
 * \param getPosition          A functor for returning the particle position.
```

we should also add a description for `lower_in_v`

Reply:
```suggestion
 * \tparam depos_order         deposition order
 * \tparam lower_in_v          lower shape order in parallel direction (Galerkin)
 * \param getPosition          A functor for returning the particle position.
```